tinyagentos/routes/shortcut_proxy.py

```
@@ -0,0 +1,488 @@
+
+    # For tui shortcuts pass the configured command; for container-terminal use
+    # the default shell (cmd=None).
+    if scope == "tui":
+        command = shortcut.get("command", "")
+        cmd: list[str] | None = ["bash", "-lc", command]
+    else:
+        cmd = None
+
+    try:
+        pty_handle = await asyncio.to_thread(_get_container_pty, agent_name, cmd)
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)